Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
```

Comment:
it's arbitrary, you're right. they don't have any tagged releases yet. as far as I could follow, in #1256 they have not decided on a solution on how to handle software that has no release versions yet. is it possible to use the commit id instead of a numeric version number?